pkg/registry/login.go

```diff
@@ -94,6 +118,58 @@ func (r RegistryAuthenticator) readDockerConfig() DockerConfig {
 	return DockerConfig{}
 }
 
+func (r RegistryAuthenticator) getHeaderOnContainerdFiles(repository string) (ContainerdServerHeader, error) {
+	directory := "/etc/containerd" // Replace with the path to the directory you want to walk
```

Comment:
shouldn't this be a parameter?

Reply:
Addressed. We still hardcode the path but at a higher level, so it should be easier to extract if it ever becomes configurable.